components/api_version/src/lib.rs

```diff
@@ -516,4 +525,41 @@ mod tests {
             }
         )
     }
+
+    #[test]
+    fn test_raw_value_encode_delete() {
```

Comment:
Suggest add unit tests to `test_no_ttl` and `test_ttl`, as test of TTL is quite similar to delete flag.
Or we can change names to `test_no_meta` and `test_meta`.

Reply:
fixed